src/components/seo.tsx

```diff
@@ -9,9 +9,17 @@ import React from "react"
 import PropTypes from "prop-types"
 import { Helmet } from "react-helmet"
 import { useStaticQuery, graphql } from "gatsby"
-import sharingImage from "../images/kintohub-og.png"
+import sharingImage from "resources/logo/share.png"
 
-function SEO({ description, lang, meta, keywords, title }) {
+type SeoProps = {
+  description: any
+  lang: any
+  meta: any
+  keywords: any
+  title: any
```

Comment:
avoid using any. you can consider using `|` if that may be as different format. e.g. `string | number | ...`